sql/sql_show.cc

```diff
@@ -9201,14 +9201,16 @@ int make_proc_old_format(THD *thd, ST_SCHEMA_TABLE *schema_table)
   Constants for columns that are present in
   SHOW ALL SLAVES STATUS
   that are not in SHOW SLAVE STATUS. Specifically, columns 0 and 1, and
-  everything at and above 56.
+  everything between 56 and 64 inclusive.
     0: Connection_name
     1: Slave_SQL_State
     56: Retried_transactions
+    64: Master_Slave_time_diff
 */
 #define SLAVE_STATUS_COL_CONNECTION_NAME 0
 #define SLAVE_STATUS_COL_SLAVE_SQL_STATE 1
 #define SLAVE_STATUS_COL_RETRIED_TRANSACTIONS 56
+#define SLAVE_STATUS_COL_MASTER_SLAVE_TIME_DIFF 64
```

Comment:
> No, I think this is wrong.

*start a thread for myself so I don’t forget*
(But yeah, something overcomplicated is goïng on with this “add at the end only” limitation.)